plugin/plugin.xml

```diff
@@ -48,4 +47,35 @@
       <arg file="${PLUGIN_OUTPUT_PROPS}"/>
     </command>
   </step-type>
+    <step-type name="Upload DSYM File">
+    <description>
+      Uploads Xcode DSYM file to NewRelic server for Desymbolication.
+	</description>
+    <properties>
+      <property name="application_id" required="true">
+        <property-ui description="New Relic mobile application id" label="Application ID" type="textBox"/>
+      </property>
+      <property name="product_name" required="true">
+        <property-ui description="Application Product Name." label="Product Name" type="textBox"/>
+      </property>
+      <property name="archive_path" required="true">
+        <property-ui description="File path to XCArchive " hidden="false" label="XCArchive path" type="textBox"/>
+      </property>
+    </properties>
+    <post-processing><![CDATA[
+        if (properties.get("exitCode") != 0) {
+            properties.put(new java.lang.String("Status"), new java.lang.String("Failure"));
+        }
+        else {
+            properties.put("Status", "Success");
+        }
+     ]]></post-processing>
+    <command program="${GROOVY_HOME}/bin/groovy">
+      <arg value="-cp"/>
+      <arg path="classes:lib/commons-beanutils-1.8.0.jar:lib/groovy-all-2.4.7.jar:lib/json-lib-2.3-jdk15.jar:lib/commons-codec-1.9.jar:lib/groovy-plugin-utils-1.0.jar:lib/nekohtml-1.9.16.jar:lib/commons-collections-3.2.1.jar:lib/gson-2.3.1.jar:lib/xercesImpl-2.9.1.jar:lib/commons-lang-2.4.jar:lib/http-builder-0.7.1.jar:lib/xml-apis-1.3.04.jar:lib/commons-logging-1.2.jar:lib/httpclient-4.5.2.jar:lib/xml-resolver-1.2.jar:lib/ezmorph-1.0.6.jar:lib/httpcore-4.4.5.jar"/>
```

Comment:
Any particular reason using ant and ivy for dependency management rather than using a Gradle script that as made available when using the community template?